sdk/mysqlflexibleservers/azure-mgmt-mysqlflexibleservers/CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
 # Release History
 
+## 1.0.0 (2025-10-22)
+
+### Features Added
+
+  - Model `MySQLManagementClient` added parameter `cloud_setting` in method `__init__`
+  - Client `MySQLManagementClient` added method `send_request`
+  - Client `MySQLManagementClient` added operation group `private_endpoint_connections`
+  - Client `MySQLManagementClient` added operation group `private_link_resources`
+  - Model `AzureADAdministrator` added property `properties`
+  - Model `Capability` added property `properties`
+  - Model `MaintenanceWindow` added property `batch_of_maintenance`
+  - Model `OperationProgressResult` added property `properties`
+  - Model `ServerBackupV2` added property `properties`
+  - Model `ServerForUpdate` added property `properties`
+  - Added model `AdministratorProperties`
+  - Added model `AdvancedThreatProtectionUpdateProperties`
+  - Added enum `BatchOfMaintenance`
+  - Added model `CapabilityPropertiesV2`
+  - Added model `MaintenancePropertiesForUpdate`
+  - Added enum `Origin`
+  - Added model `PrivateEndpointConnectionListResult`
+  - Added model `PrivateLinkResource`
+  - Added model `PrivateLinkResourceProperties`
+  - Added model `ServerBackupPropertiesV2`
+  - Added model `ServerPropertiesForUpdate`
+  - Added model `ValidateBackupResponseProperties`
+  - Added operation group `PrivateEndpointConnectionsOperations`
+  - Added operation group `PrivateLinkResourcesOperations`
+
+### Breaking Changes
+
+  - This version introduces new hybrid models which have dual dictionary and model nature. And please follow https://aka.ms/azsdk/python/migrate/hybrid-models for migration.
+  - Model `AdvancedThreatProtectionForUpdate` deleted or renamed its instance variable `state`
+  - Model `AzureADAdministrator` instance variables `administrator_type`, `login`, `sid`, `tenant_id`, and `identity_resource_id` have been moved under property `properties`
+  - Model `Capability` instance variables `supported_geo_backup_regions`, `supported_flexible_server_editions`, `supported_server_versions`, and `supported_features` have been moved under property `properties`
+  - Model `MaintenanceUpdate` deleted or renamed its instance variable `maintenance_start_time`
+  - Model `OperationProgressResult` instance variable `object_type` has been moved under property `properties`
+  - Model `ServerBackupV2` instance variables `backup_name_v2`, `backup_type`, `completed_time`, `source`, and `provisioning_state` have been moved under property `properties`
+  - Model `ServerForUpdate` instance variables `administrator_login_password`, `version`, `storage`, `backup`, `high_availability`, `maintenance_policy`, `maintenance_window`, `replication_role`, `data_encryption`, and `network` have been moved under property `properties`
+  - Model `ValidateBackupResponse` deleted or renamed its instance variable `number_of_containers`
```

Comment:
@ChenxiJiang333 Let us simplify the changelog like https://github.com/Azure/azure-sdk-for-python/pull/43559/commits/a9998881cb3a08b62edbc56866074da5f27eb135